test/e2e/app_change_test.go

```diff
@@ -89,3 +93,112 @@ spec:
 		require.Equal(t, 2, len(resp2.Tables[0].Rows), "Expected to have 2 app-changes")
 	})
 }
+
+func TestAppKindChangeWithMetadataOutput(t *testing.T) {
+	env := BuildEnv(t)
+	logger := Logger{}
+	kapp := Kapp{t, env.Namespace, env.KappBinaryPath, logger}
+
+	yaml1 := `
```

Comment:
Not a blocker per say, but in interest of keeping the tests minimal 🤔 

We could have a ConfigMap and a Secret instead of a Deployment and a Service right? 
I think it would be less verbose and probably make the test faster